### PR TITLE
Use simpler regexes in [appveyortests] to address lgtm alert

### PR DESCRIPTION
**Pull request by paulmelnikow**
Two of these regexes have triggered a LGTM alert.

https://lgtm.com/rules/1505904457770/

I’m not terribly concerned about it given this is a test, though it’s nice to clear these up, and the new regexes are a bit easier to understand.

### CODE REVIEW

**Comment by paulmelnikow** [comment]
This pull request **fixes 2 alerts** when merging affa2a42d8efc167a58dbc04173764bedd9be3f0 into a58de5428140024429970ddbaf09d10906fcbe9e - [view on LGTM.com](https://lgtm.com/projects/g/badges/shields/rev/pr-3e159ec33f6da7e9ca5cdc9a38075e30c09cdcb4)

**fixed alerts:**

* 2 for Inefficient regular expression

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: services/appveyor/appveyor-tests.tester.js [hunk 1]
```diff
@@ -3,19 +3,19 @@
 const Joi = require('joi')
 
 const isAppveyorTestTotals = Joi.string().regex(
-  /^(?:[0-9]+ (?:passed|skipped|failed)(?:, )?)+$/
+  /^[0-9]+ passed(, [0-9]+ failed)?( [0-9]+ skipped)?$/
```
Comment (PyvesB) [inline services/appveyor/appveyor-tests.tester.js]: Isn't the comma separator missing for `skipped`?

Comment (paulmelnikow) [reply services/appveyor/appveyor-tests.tester.js]: Indeed! Good spot!

**Comment by paulmelnikow** [comment]
This pull request **fixes 2 alerts** when merging 7c7edc2e31bbd949e0aea700f067c2e53a648ed5 into 79af2840f3701301ab69d0ba2000273b512f0dc5 - [view on LGTM.com](https://lgtm.com/projects/g/badges/shields/rev/pr-e5a93550fdaf8c7c3c09bc087add58c7278f89c4)

**fixed alerts:**

* 2 for Inefficient regular expression

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by paulmelnikow** [comment]
This pull request **fixes 2 alerts** when merging 6df4e52f34424b880e23a45838dbd1982d794a39 into 55ce947a35a0b32075fcdf7836785fb8b180c7d1 - [view on LGTM.com](https://lgtm.com/projects/g/badges/shields/rev/pr-f0a8b1a45b805ca3256e6acac68285d784a41eb5)

**fixed alerts:**

* 2 for Inefficient regular expression

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**